pkg/kube/wait.go

```diff
@@ -44,7 +45,10 @@ import (
 type waiter struct {
 	c       kubernetes.Interface
 	timeout time.Duration
-	log     func(string, ...interface{})
+}
+
+func (w *waiter) log(format string, v ...interface{}) {
+	fmt.Fprintf(os.Stdout, fmt.Sprintf("%s\n", format), v...)
```

Comment:
informational messages should be displayed on `os.Stderr` so that it can be stripped from the output.